src/transform3d.rs

```diff
@@ -177,6 +177,17 @@ where T: Copy + Clone +
         )
     }
 
+    /// Checks if we're facing backface
```

Comment:
The comment is a bit confusing because the matrix itself does not face any direction. Could you replace it with something like "Check whether shapes on the XY plane with Z pointing towards the screen transformed by this matrix would be facing back."